tests/apidocs/util.py

```diff
@@ -22,9 +23,25 @@ def create_schema(self):
             return create_spec(data)
 
     def validate_schema(self, request, response):
+        assert 200 <= response.status_code < 300, response.status_code
+
+        response["Content-Type"] = "application/json"
         result = ResponseValidator(self.create_schema()).validate(
             DjangoOpenAPIRequest(request), DjangoOpenAPIResponse(response)
         )
 
         result.raise_for_errors()
         assert result.errors == []
+
+    def create_event(self, name, **kwargs):
```

Comment:
we used to have a test factory method that was called `create_event` that was pre `store_event` but it looks like that actually got removed so this should be okay.